dhall-docs/doctest/Main.hs

```diff
@@ -0,0 +1,20 @@
+module Main where
+
+import System.FilePath ((</>))
+
+import qualified GHC.IO.Encoding
+import qualified System.Directory
+import qualified System.IO
+import qualified Test.DocTest
+
+main :: IO ()
+main = do
+    GHC.IO.Encoding.setLocaleEncoding System.IO.utf8
+    pwd    <- System.Directory.getCurrentDirectory
+    prefix <- System.Directory.makeAbsolute pwd
+    Test.DocTest.doctest
+        [ "--fast"
+        , "-XOverloadedStrings"
+        , "-XRecordWildCards"
```

Comment:
Usually you want to add `X`-flags at the top of files instead of the compiler invocation, so that tooling can figure it out without context.